newt/repo/repo.go

```diff
@@ -198,7 +198,11 @@ func (r *Repo) Name() string {
 }
 
 func (r *Repo) Path() string {
-	return r.localPath
+	if r != nil {
+		return r.localPath
+	} else {
+		return ""
+	}
```

Comment:
This will "hide" all issues where `Path()` is called unintentionally on `nil` object, so better to actually explicitly handle this on caller side. Of course as long as there is some valid case in which we can actually have `nil` repo that we should handle properly, not sure when this could happen though tbh.